bindings_generator/src/api.rs

```diff
@@ -257,15 +324,24 @@ impl Ty {
             "enum.Variant::Type" => Ty::VariantType,
             "enum.Variant::Operator" => Ty::VariantOperator,
             ty if ty.starts_with("enum.") => {
+                // Enums may reference known types (above list), check if it's a known type first
                 let mut split = ty[5..].split("::");
-                let mut class = split.next().unwrap();
-                if class.starts_with('_') {
-                    class = &class[1..];
+                let class_name = split.next().unwrap();
```

Comment:
Is the underscore stripping no longer necessary here?

Reply:
Lower, the call `to_snake_case()` takes care of it.